src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_NUMBER = 5;
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = "WHITE";    
```

Comment:
```suggestion
    private static final String DEFAULT_COLOR = Color.WHITE;    
```

Reply:
It doesn`t work 
![image](https://user-images.githubusercontent.com/101352509/185944495-98776742-4649-4f47-a27e-88ff5280618e.png)

Reply:
Why isn`t it right? I saw an approved solutions with "private static final String DEFAULT_COLOR = "WHITE""

Reply:
change type to Color

Reply:
Thanks, I did it. Is it completed?